### PR TITLE
20231018 Update Flashforge printer profiles: Adventurer 5M and Adventurer 5M Pro

### PR DESCRIPTION
**Pull request by FlashforgeOfficial**
Hi, 
Lately I tested the profiles on a local clone of Orca. The printer profiles can now be selected and loaded with success. 
Let me know if the PR still doesn't match the main software.  Thanks!

### CODE REVIEW

File: resources/profiles/Flashforge/.vs/Flashforge/FileContentIndex/3f995606-d940-48a3-b978-2919ce9135e5.vsidx
Comment (SoftFever) [inline resources/profiles/Flashforge/.vs/Flashforge/FileContentIndex/3f995606-d940-48a3-b978-2919ce9135e5.vsidx:1]: Can you remove .vs folder?

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by SoftFever** [comment]
@FlashforgeOfficial 
Thanks. The profiles can be loaded properly.

I noticed some potential issues in the profiles though:
1. The origin is set in center of the bed, this may or may not be a bug. But it's unusual to have origin in the center for non-delta printer. Please double check.
2. The machine is using absolute E distance instead of recommended relative E distance. We need to reset E distance in layer change gcode for absolute  E distance. Slice a cube you will see the errors.
3. It's recommended to use transparent background color for the bed texture

**Comment by RafBois** [comment]
Hello, can I get a profile for the Guider3Plus? 

The FlashForge slicer (FlashPrint) is just impossible to use (I'm on a Mac) with a room that takes up the volume of this printer.
The settings options for the FlashPrint slicer are extremely limited.
The machine lacks a lot of options to be considered a professional machine.

File: resources/profiles/Flashforge/.vs/Flashforge/FileContentIndex/3f995606-d940-48a3-b978-2919ce9135e5.vsidx
Comment (FlashforgeOfficial) [reply resources/profiles/Flashforge/.vs/Flashforge/FileContentIndex/3f995606-d940-48a3-b978-2919ce9135e5.vsidx:1]: Sure, this folder will be removed in the next PR.

**Comment by FlashforgeOfficial** [comment]
> @FlashforgeOfficial Thanks, the profiles loaded correctly. I did notice some potential issues however, can you confirm?
> 
> 1. The origin is in the bed's center. It's rare for non-delta printers to have this setting. Can you double-check?
> 2. The machine uses absolute E distance, not the recommended relative E distance. With absolute E distance, we must reset the E distance in the layer change gcode. Slice a cube, and you'll see the errors.
> 3. It's recommended to use a transparent background for the bed texture.

Yes, for tip1 the Flashforge printers use the bed's center for origin by default so we will keep the setting unless something goes wrong. For tips 2 and 3 we will double check and make the improvements. Thank you!

**Comment by FlashforgeOfficial** [comment]
> Hello, can I get a profile for the Guider3Plus?
> 
> The FlashForge slicer (FlashPrint) is just impossible to use (I'm on a Mac) with a room that takes up the volume of this printer. The settings options for the FlashPrint slicer are extremely limited. The machine lacks a lot of options to be considered a professional machine.

Hi 

> Hello, can I get a profile for the Guider3Plus?
> 
> The FlashForge slicer (FlashPrint) is just impossible to use (I'm on a Mac) with a room that takes up the volume of this printer. The settings options for the FlashPrint slicer are extremely limited. The machine lacks a lot of options to be considered a professional machine.

Thank you @RafBois. Try using newest version of FlashPrint as there are some nice feature updates and might solve the issues. It is not clear if orca profile for G3 series would be setup for now, yet we will consider adding support for more machine profiles.

**Comment by RafBois** [comment]
You can't even read change logs. It's already half-developed. Frankly above FlashForge. I don't understand why they want to close everything with their software? What's in it for them?
Capture d’écran 2023-10-23 à 17 42 07

Best regards

File: resources/profiles/Flashforge/machine/Flashforge Adventurer 5M Pro 0.4 Nozzle.json
Comment (mariuske) [inline resources/profiles/Flashforge/machine/Flashforge Adventurer 5M Pro 0.4 Nozzle.json:1]: @FlashforgeOfficial - It looks like the nozzle specific parameters are copied from the 0.6 nozzle file and not adjusted for a 0.4 Nozzle. Examples (might be others as well):
	"nozzle_diameter": [ "0.6" ],
	"printer_variant": "0.6",
	"z_hop": [ "0.6" ],

Comment (FlashforgeOfficial) [reply resources/profiles/Flashforge/machine/Flashforge Adventurer 5M Pro 0.4 Nozzle.json:1]: Thank you for the advice!  I am fixing them in the lastest PR.

**Pull request closed without merging**